modules/installation-special-config-storage.adoc

```diff
@@ -54,21 +54,22 @@ metadata:
   labels:
     machineconfiguration.openshift.io/role: worker
 boot_device:
-  layout: x86_64
+  layout: x86_64 <1>
   luks:
-    tpm2: true <1>
-    tang: <2>
+    tpm2: true <2>
+    tang: <3>
       - url: http://tang1.example.com:7500
         thumbprint: jwGN5tRFK-kF6pIX89ssF3khxxX
       - url: http://tang2.example.com:7500
         thumbprint: VCJsvZFjBSIHSldw78rOrq7h2ZF
-    threshold: 2 <3>
+    threshold: 2 <4>
 openshift:
   fips: true
 ----
-<1> Include this field if you want to use a Trusted Platform Module (TPM) to encrypt the root file system.
-<2> Include this section if you want to use one or more Tang servers.
-<3> Specify the minimum number of TPM v2 and Tang encryption conditions that must be met for decryption to occur.
+<1> On ppc64le nodes, set this field to `ppc64le`. On arm64 based nodes, set this field to `aarch64`. On all other nodes, this field can be omitted.
```

Comment:
Hi @kelbrown20, I know 'examples' are not well appreciated in the doc, but the following is to try giving you the concept I'd like to expose here:

```
Set the <arch> field to the instruction set architecture of the cluster nodes: e.g., x86_64, aarch64, ppc64le...
```

with `layour: <arch>` at row 57.

Reply:
I'm a bit concerned about how to express that we need the GNU way of expressing the architecture (aarch64, and not arm64) here, while most of the other parts of the docs use the Golang/llvm way (arm64, amd64).